### ISSUE TITLE
Feature Request: Continent Grouping and States Drilldown

### ISSUE THREAD

**Issue by jburse** [CSSEGISandData/COVID-19#1716]
The list of counries/regions is already 177 items long. Would it possibly to provide?

- **Continent Grouping:**
  Like for example have a meta item "Europe" that can be collapsed/expanded,
  which would house all Europe countries/regions. Other continents might
  get the same treatment.
- **States Drilldown:**¨
  Like for example the USA states as country/regions but again grouped
  into "USA" with collapse/expand control. Other big countries with
  substates might get the same treatment.

Just currious. Also I don't know whether this feature request is a duplicate.
Maybe #862 is similar.

**Comment by jburse** [comment]
Germany has a solution with two different dashboards:
"Bundesländer" and "Landkreise"
https://experience.arcgis.com/experience/478220a4c454480e823b17327b2bf1d4

So maybe my feature request might be rephrased is there
a Continents dashboard somewhere?

**Comment by cipriancraciun** [comment]
@jburse I have derived a dataset which does the groupings you are asking for:
* https://github.com/cipriancraciun/covid19-datasets
* I have described it in issue #1281 or the readme in the repo above;

**Comment by jburse** [comment]
Ok I see, this one:

https://github.com/cipriancraciun/covid19-datasets/tree/master/plots/jhu/continents

**Comment by cipriancraciun** [comment]
@jburse yes, I updated it multiple times a day.  (Although the original upstream JHU dataset is updated only once a day, thus even if I updated it multiple times a day, I still depend on the JHU dataset.)

**Comment by jburse** [comment]
America, includes north and south america? How about splitting it?
Europe, includes russia siberia and non-siberia?
How about including Asia (without China), Africa and Australia?

![Continental_models-Australia](https://user-images.githubusercontent.com/5411487/77833977-9a137200-7141-11ea-90a4-8e979b819e79.gif)

Sorry for some many requests.

**Comment by cipriancraciun** [comment]
@jburse in my datasets you have two levels: continents and sub-continents.  Thus you can choose the granularity you want.

The rows with `location_type == "total-region"` includes:
~~~~
Africa
Americas
Asia
Europe
Oceania
~~~~

The rows with `location_type == "total-subregion"` includes:
~~~~
Australia and New Zealand
Caribbean
Central America
Central Asia
Central Europe
Eastern Africa
Eastern Asia
Eastern Europe
Melanesia
Micronesia
Middle Africa
North America
Northern Africa
Northern Europe
South America
South-Eastern Asia
Southern Africa
Southern Asia
Southern Europe
Western Africa
Western Asia
Western Europe
~~~~

**Comment by jburse** [comment]
Looks like I am interested in something inbetween.

**Comment by cipriancraciun** [comment]
@jburse I'll analyze this tomorrow and see if I can include some more regions.

Would you want to open a ticket in my repository about this?  (I don't want to clutter JHU's repository with things that aren't that much related with theirs repository).
* https://github.com/cipriancraciun/covid19-datasets

So if you have suggestions on how to group the sub-continents, please group them and I'll see.

**Comment by jburse** [comment]
I am still thinking about where another grouping might come from.
I think there is a certain economic viewpoint involved. If I find a

As soon as paper or something detailing another grouping,  I 
might open a ticket at your repository. Please be patient.

**Comment by grwlf** [comment]
May be related: Here is an unofficial clone containing details on Russia regions https://github.com/grwlf/COVID-19_plus_Russia

**Comment by jburse** [comment]
A friend of mine sent me this. Still trying 
to figure out, which website it is:

![20200331_101010_Corona_welt_SEA_k1](https://user-images.githubusercontent.com/5411487/78491639-f3366380-7748-11ea-9583-08e876e8e8c8.jpg)
